### PR TITLE
Add token to session

### PR DESCRIPTION
**Pull request by InouRidder**
DONE:

Add token to session to generate URL to Review-ui on Client-ui.
Respond with full error backtrace in production for debug in prod (TEMP)
Add default_url[:host] for the method `poymorphic_url`

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -39,6 +39,10 @@ def user_not_authorized(exception)
     }, status: :unauthorized
   end
 
+  def default_url_options
```
Comment (DanielGordon1) [inline app/controllers/application_controller.rb]: Are you sure this is necesarry?

File: config/environments/production.rb [hunk 2]
```diff
@@ -1,9 +1,14 @@
+# frozen_string_literal: true
+
+Rails.application.routes.default_url_options[:host] = 'https://tablevibes-backend-n56jnry35q-ew.a.run.app'
```
Comment (DanielGordon1) [inline config/environments/production.rb]: I would guess that this would be sufficient

File: app/models/answer.rb [hunk 3]
```diff
@@ -25,6 +27,13 @@ def found_us_through_answer?
 
   private
 
+  def normalize_subject
```
Comment (DanielGordon1) [inline app/models/answer.rb]: As we discussed; not a big fan of this :p

File: db/seeds/development.rb [hunk 4]
```diff
@@ -141,7 +141,7 @@
 puts 'Adding some gifts and conditions'
 
 gift1 = Gift.create!(
-  distributer: venue1,
+  distributer: account1,
```
Comment (DanielGordon1) [inline db/seeds/development.rb]: 👍

File: app/models/answer.rb (hunk 3 above)
Comment (InouRidder) [reply app/models/answer.rb]: The other solution would require much more time.

File: config/environments/production.rb (hunk 2 above)
Comment (InouRidder) [reply config/environments/production.rb]: yep. forgot to remove one of them.

Comment (InouRidder) [reply config/environments/production.rb]: fixed in 
dfb3bc7

**Pull request merged**